src/Sulu/Component/Persistence/EventSubscriber/ORM/UserBlameSubscriber.php

```diff
@@ -123,6 +123,8 @@ public function onFlush(OnFlushEventArgs $event)
 
     private function handleUserBlame(OnFlushEventArgs $event, UserInterface $user, bool $insertions)
     {
+        // TODO: calling getEntityManager on the event is deprecated, but access to the UnitOfWork is not
+        // guaranteed for an ObjectManager. So we need to get an EntityManager somehow different.
```

Comment:
I suggest using `getObjectManager` and then making a check about the type of manager you receive

Reply:
Thanks. This is done.